tutorial/docs/quickstart/dashboard-quickstart.mdx

```diff
@@ -0,0 +1,137 @@
+![Peer copied](./img/dashboard_peer_copied.png)
+
+To connect the peer to the room, you can either press the **Connect** button in the peer's row, or scan the QR code with your mobile app, and connect there.
+
+### Sending tracks
+
+When the peer connects to the room, you will see the track menu. It allows you to send tracks to the room.
+![Track menu](./img/dashboard_track_menu.png)
+
+You can select the _audio or video tracks_ You want to send to the room. Apart from your camera/microphone,
+there is an option to pick one from a list of _mock tracks_ that we prepared.
+
+You can also select the _resolution_ of the video track
+or if you want, send the video as a _simulcast_ stream.
+
+There is also an option to add custom _metadata_ (in JSON format) to the track. The peer will send metadata with the track to the room. Each recipient will be able to see it.
+
+When you select the track, click the **Add track** button. If everything is correct, client will send the track to the room and other peers
+will be able to see and hear you.
+
+### Receiving tracks
+
+When the peer connects to the room, it will automatically receive tracks from other peers.
+
+#### Congrats! The basic utils of the dashboard are now covered. Next, we will dive into useful features that will help you with testing.
+
+## A ton of information that the dashboard provides
+
+### Server logs in console
+
+On the side menu, you can set up the dashboard to show the server logs in the console.
+Logs are great for debugging purposes and present the variety of communication that takes place between the dashboard and the backend.
+You can also register your callbacks in your Jellyfish projects.
+![Server logs](./img/dashboard_server_logs.png)
+
+### Server/Room/Client state
+
+Each server, room, and client has its state. You can see the state of each of them in the dashboard by clicking appropriate buttons.
+![Server state](./img/dashboard_states.png)
+
+### Server events
+
+Apart from the state, you can also see the events that occur on the server.
+You can see them after clicking the **Show Server Events** button.
+
+![Server events](./img/dashboard_server_events.png)
+
+## HLS and RTSP streaming
+
+The dashboard allows adding HLS and RTSP streams in the room. You can set up the streams and see them in the room. It will look like this:
+![HLS and RTSP](./img/dashboard_hls_rtsp.png)
+
+## Loadbalancing in Dashboard
+
+Jellyfish Server supports loadbalancing. It means that you can run multiple instances of the server and use them simultanoesly to minimise the load on each of them.
+The dashboard reflects this feature. Provided you run multiple-in-one Jellyfish instance (docker does that by default),
+connect one server to the dashboard. When the room you created was moved to another server, the dashboard will open this server for you.
```

Comment:
> When the room you created was moved to another server, the dashboard will open this server for you

That could be misleading. The room was not moved, but it was created on another server